docs/documentation.md

```diff
@@ -1 +1,69 @@
-# PURIS Frontend Documentation
\ No newline at end of file
+# PURIS Frontend Documentation
+
+## Frontend Installation
+
+### Requirements
+To use the PURIS frontend, you need:
+
+- a running [PURIS backend](https://github.com/eclipse-tractusx/puris-backend/)
+- a running [Product EDC](https://github.com/catenax-ng/product-edc)
+- a running [CX backend service](https://github.com/denisneuling/cx-backend-service)
```

Comment:
Pretty unsure if this is ok @Siegfriedk @SebastianBezold @almadigabor. See also Line 69 in this file.

Reply:
I also don't understand the intent of this cx-backend-service, checked the repo, Dockerfile, chart but I think it shouldn't be referenced in an eclipse-tractusx repo.

Reply:
I think you can reference it but the implication of it makes it potentially an issue: If PURIS requires NON eclipse foundation project (espeically the denisneuling one), you are not contributing all necessary product components of PURIS to eclpse foundation.

Reply:
The PURIS application uses the EDC (to be more concrete, the [product-edc](https://github.com/catenax-ng/product-edc) / [tractusx-edc](https://github.com/eclipse-tractusx/tractusx-edc)).

@almadigabor : You mentioned that:
> [...] I think it shouldn't be referenced in an eclipse-tractusx repo.

Which is in line with @Siegfriedk s comment:
> If PURIS requires NON eclipse foundation project (espeically the denisneuling one), you are not contributing all necessary product components of PURIS to eclpse foundation.

As mentioned, we are using the EDC. And the EDC itself in the latest version in the Eclipse Tractus-X space uses components not part of the eclipse foundation.
See:

- https://github.com/eclipse-tractusx/tractusx-edc/blob/main/edc-tests/src/main/resources/deployment/helm/all-in-one/Chart.yaml#L48
- https://github.com/eclipse-tractusx/tractusx-edc/blob/main/edc-tests/src/main/resources/deployment/helm/all-in-one/Chart.yaml#L75

Therefore, the problem isn't related to this PURIS application directly but more or less an upstream problem in the tractusx-edc.

Reply:
Since the "CX backend service", we will update this in a newer version!